templates/workspace_index/index.html.j2

```diff
@@ -24,4 +24,8 @@
         </li>
     {% endfor %}
 </ul>
-{% endblock %}
\ No newline at end of file
+
+{% if page.docs_content %}
```

Comment:
I believe we want this to sit above the workspace members (with the preferred members, people can still specify which members to put _above_ the content)